src/target.cr

```diff
@@ -0,0 +1,17 @@
+module Shards
+  class Target
+    getter name : String
+    property main : String
+    property options : Array(String)
+
+    def initialize(@name)
+      super()
+      @main = ""
+      @options = [] of String
+    end
+
+    def cmd
```

Comment:
What about `shell_command`? Thought the model shouldn't have to deal with the invocation. Command::Build should be the one generating the command.